src/components/teasers/teaser.stories.tsx

```diff
@@ -1,16 +1,11 @@
 import React from 'react';
-import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { ComponentStory } from '@storybook/react';
 import { Teaser } from './teaser';
 import ExampleImage from '../../../blog-posts/images/angular.png';
 import styled from 'styled-components';
 import { Illustration, IllustrationSize } from '../illustration/illustration';
 import { ILLUSTRATION_NAMES } from '../illustration/illustration-set';
 
-export default {
-  component: Teaser,
-  title: 'Components/Teaser',
-} as ComponentMeta<typeof Teaser>;
-
 const Template: ComponentStory<typeof Teaser> = (args) => <Teaser {...args} />;
 
 export const Regular = Template.bind({});
```

Comment:
Even though it's not part of this PR: Would you mind changing the default value for the `title` property in these stories to `Teaser title` instead of `Leadbox title`?

Reply:
I've already changed it in https://github.com/satellytes/satellytes.com/pull/198. Thanks anyway 👍